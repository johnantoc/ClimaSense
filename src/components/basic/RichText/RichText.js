/**
 * Rich Text Component.
 */

import React from "react";
import { Text } from "react-native";

import RichTextStyles from "./RichText.style";

const RichText = ({ text, styles }) => {
  return <Text style={[RichTextStyles.text, styles]}>{text}</Text>;
};

export default RichText;
