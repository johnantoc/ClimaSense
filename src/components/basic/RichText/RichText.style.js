import { StyleSheet } from "react-native";

const RichTextStyles = StyleSheet.create({
  text: {
    color: "#000119",
    fontFamily: "roboto-regular",
  },
});

export default RichTextStyles;
