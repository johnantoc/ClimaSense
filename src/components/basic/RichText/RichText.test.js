import React from 'react';
import RichText from './RichText';

import renderer from 'react-test-renderer';

test('RichText renders correctly', () => {
  const tree = renderer.create(<RichText />).toJSON();
  expect(tree).toMatchSnapshot();
});
