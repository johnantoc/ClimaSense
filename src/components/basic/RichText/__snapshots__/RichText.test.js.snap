// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RichText renders correctly 1`] = `
<Text
  style={
    Array [
      Object {
        "color": "black",
        "justifyContent": "center",
      },
    ]
  }
>
  Hello World
</Text>
`;
